app/controllers/appointments_controller.rb

```diff
@@ -16,21 +16,13 @@ def index
       @search_filters = DEFAULT_SEARCH_FILTERS
     end
 
-    scope = PatientSummary.where(next_appointment_facility_id: current_admin.accessible_facilities(:manage_overdue_list))
-
-    @patient_summaries = PatientSummaryQuery.call(relation: scope, filters: @search_filters)
-
-    if current_facility
-      @patient_summaries = @patient_summaries.where(assigned_facility_id: current_facility.id)
-    end
-    @patient_summaries = @patient_summaries.order(risk_level: :desc, next_appointment_scheduled_date: :desc, id: :asc)
-
     respond_to do |format|
       format.html do
-        @patient_summaries = paginate(@patient_summaries).includes(:next_appointment, patient: :appointments)
+        @patient_summaries = paginate(patient_summaries(only_overdue: true)).includes(:next_appointment, patient: :appointments)
```

Comment:
`only_overdue` seems like a weird name for a flag that generates overdue lists 🤔 I really think we should pursue the idea of `requires_reminder`, `requires_call`, `due_for_call`, or something like that. In plain English, that's a fairly intuitive way to describe the difference between the dashboard and download, right? The dashboard only shows people you need to call, the download shows you everyone?

Reply:
Yeah, that's pretty much the main difference between dashboard and download. Maybe `all_patients_requiring_call` or `all_patients_requiring_follow_up` would be better here. What about `only_scheduled_overdue`?